packages/boilerplate-generator/boilerplate_web_browser_template.js

```diff
@@ -0,0 +1,70 @@
+// Template function for rendering the boilerplate html for browsers
+// Replicates the template defined in boilerplate_web.browser.html
+// Arguments: root : { htmlAttributes, css : [{ url }], bundledJsCssUrlRewriteHook : Function, head, dynamicHead, body, dynamicBody, inlineScriptsAllowed, additionalStaticJs, meteorRuntimeConfig }
+
+export default function (manifest) {
+  const root = manifest;
```

Comment:
Not sure why you're renaming the parameter instead of just giving it whatever name you want it to have.